ratdb/OPTICS_Photocathode.ratdb

```diff
@@ -291,4 +291,59 @@ THICKNESS_value2: [26e-6, 26e-6, ],
 PROPERTY_LIST: ["EFFICIENCY", "RINDEX", "KINDEX", "THICKNESS", ]
 
 
+}
+
+{
+// E.Keener at Penn, copied from photocathode_R7081HQE, quantum efficiency data from R7081-20 datasheet
+name: "OPTICS",
+index: "photocathode_R7081",
+valid_begin : [0, 0],
+valid_end : [0, 0],
+photocathode: 1,
+
+
+EFFICIENCY_option: "wavelength",
+EFFICIENCY_value1: [277.03253918, 279.28433418, 280.42931468, 281.57429519, 283.82768043,
+                    283.85630495, 286.1160512 , 286.14467571, 288.40124146, 291.76780219,
+                    296.24912868, 300.73681616, 302.99020141, 306.35835239, 308.61650839,
+                    311.98465938, 317.58075161, 328.73795054, 344.34308069, 364.38978105,
+                    387.76964688, 407.80680574, 428.95395959, 450.09793294, 469.01396505,
+                    489.03999215, 506.83966827, 524.63139314, 541.30835226, 553.53260942,
+                    566.86368106, 582.42428419, 594.64536084, 605.75167175, 614.63163168,
+                    621.28206012, 629.04566405, 635.69768274, 644.57287191, 652.33647585,
+                    657.87849955, 663.40939149, 667.83664945, 674.48548764, 677.79479935,
+                    683.3288718 , 687.75612975, 693.28861195, 699.93904039, 704.36311785,
+                    709.90196104]
+
+EFFICIENCY_value2: [1.01037329e-03, 1.31460227e-03, 1.82669126e-03, 2.53825894e-03, 3.35730906e-03,
+                    4.51401598e-03, 6.37661202e-03, 8.57357128e-03, 1.17193335e-02, 1.55014228e-02,
+                    2.08447332e-02, 2.99359208e-02, 3.95956995e-02, 5.32426355e-02, 7.39849817e-02,
+                    9.94844254e-02, 1.35999090e-01, 1.76992368e-01, 2.19279210e-01, 2.42154895e-01,
+                    2.50413246e-01, 2.50550824e-01, 2.42583844e-01, 2.27270025e-01, 2.09436834e-01,
+                    1.86763270e-01, 1.66534175e-01, 1.36773163e-01, 1.10494913e-01, 8.92546121e-02,
+                    6.75088638e-02, 5.27719554e-02, 4.12482776e-02, 3.17140950e-02, 2.43821681e-02,
+                    1.81376102e-02, 1.34927682e-02, 1.02035590e-02, 7.46694670e-03, 5.55474397e-03,
+                    4.34097381e-03, 3.02350125e-03, 2.32421952e-03, 1.70075600e-03, 1.24442040e-03,
+                    8.95727402e-04, 6.88561683e-04, 4.87538273e-04, 3.62674030e-04, 2.69772513e-04,
+                    2.04002300e-04]
+
+RINDEX_option: "wavelength",
+RINDEX_value1: [60.0, 200.0, 380.0, 395.0, 410.0, 425.0, 440.0, 455.0, 470.0, 485.0, 500.0, 515.0, 530.0, 545.0, 560.0, 575.0, 590.0, 605.0, 620.0, 635.0, 650.0, 665.0, 680.0, 800.0, ],
+RINDEX_value2: [1.0, 1.0, 1.92, 2.18, 2.38, 2.61, 2.70, 2.87, 3.00, 3.00, 3.00, 3.09, 3.26, 3.20, 3.12, 3.06, 3.01, 2.98, 2.96, 2.95, 2.95, 2.95, 2.96, 3.0, ],
+
+
+//RINDEX_option: "wavelength",
```

Comment:
Can you remove the commented out code (I suppose for all cases, in this file).

Reply:
Fixed in latest commit